packages/search/addon/components/navi-search-result/definition.hbs

```diff
@@ -1,24 +1,33 @@
 {{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
 <table class="navi-search-results__item navi-search-results__item--definition" ...attributes>
   <tbody class="navi-search-results__item-body">
-    {{#each @data as |result|}}
-      <tr class="navi-search-definition-result">
-        <td class="navi-search-definition-result__item navi-search-results__item--cell">
-            <h5 class="navi-search-definition-result__item-name">{{result.name}}</h5>
-            {{#if this.hasMultipleDataSources}}
-              <em class="navi-search-definition-result__item-source">Source: {{result.source}}</em>
-            {{/if}}
-            <p class="navi-search-definition-result__item-description">
-              {{#if result.description}}
-               {{result.description}}
-              {{else}}
-                {{#let (await result.extended) as |extendedResult|}}
-                  {{extendedResult.description}}
-                {{/let}}
-              {{/if}}
-            </p>
+    {{#each this.results as |result|}}
+      {{#let (await result.extended) as |extendedResult|}}
+        {{#if (or result.description extendedResult.description)}}
+          <tr class="navi-search-definition-result">
+            <td class="navi-search-definition-result__item navi-search-results__item--cell">
+                <h5 class="navi-search-definition-result__item-name">{{result.name}}</h5>
+                {{#if this.hasMultipleDataSources}}
+                  <em class="navi-search-definition-result__item-source">Source: {{result.source}}</em>
+                {{/if}}
+                <p class="navi-search-definition-result__item-description">
+                  {{#if result.description}}
+                  {{result.description}}
+                  {{else}}
+                    {{extendedResult.description}}
+                  {{/if}}
+                </p>
+            </td>
+          </tr>
+        {{/if}}
+      {{/let}}
+    {{/each}}
+    {{#if this.hasMoreResults}}
+      <tr {{on "click" (toggle "showTop" this)}} class="navi-search-definition-options">
```

Comment:
Can we use the text button here: https://denali.design/docs/2/components/buttons

Also does this need to be in the table?

Reply:
I felt the button stood out too much in the results that's why I used the chip styles.

Also yea I'll move it outside of the table.

Reply:
Denali `text` button will not stand out as it is just text

Reply:
> Chips are used to highlight important text information
https://denali.design/docs/2/components/chips

This is an interactive element, we should use a button